13-erc20-staking/scripts/mint.js

```diff
@@ -0,0 +1,17 @@
+const { expect } = require('chai');
+
+async function main() {
+  const [owner, addr1] = await ethers.getSigners();
+  const initialSupply = '100000000000000000000';
+  const myTokenContract = await await hre.ethers.deployContract("MyToken", [initialSupply, owner.address]);
+  await myTokenContract.waitForDeployment();
+  await myTokenContract.connect(owner).mint(addr1.address, 1000);
```

Comment:
try to mint amount at least equivalent to 1 ETH or more

Reply:
is it okay to use gwei instead to save some eth.